README.adoc

```diff
@@ -20,7 +20,7 @@ A new HiveMQ extension project can be created by running the following maven com
 
 [source,bash]
 ----
-mvn archetype:generate -DarchetypeGroupId=com.hivemq -DarchetypeArtifactId=hivemq-extension-archetype -DarchetypeVersion=4.6.2
+mvn archetype:generate -DarchetypeGroupId=com.hivemq -DarchetypeArtifactId=hivemq-extension-archetype -DarchetypeVersion=4.7.0 -DhivemqExtensionSdkVersion=4.6.2
```

Comment:
i think we said in the standup that we can not make anything publish that has 4.7.0 in its name?